### PR TITLE
Fix issues in SCIM attribute schema

### PR DESCRIPTION
**Pull request by kayathiri4**
### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by yasinmiran — APPROVED** [review]

**Review by DimalChandrasiri — APPROVED** [review]

File: apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx [hunk 1]
```diff
@@ -167,6 +167,32 @@ export const EditExternalClaim: FunctionComponent<EditExternalClaimsPropsInterfa
         }
     }, [ externalClaims, localClaims, claim, addedClaim ]);
 
+    /**
+     * Get the attribute name.
+     */
+    const resolveClaimURIName = (): string => {
+        const parts = addedClaim.claimURI.split(":");
```
Comment (brionmario) [inline apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx:174]: Let's add types.

```
const parts: string[] = addedClaim.claimURI.split(":");
```

File: apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx [hunk 2]
```diff
@@ -167,6 +167,32 @@ export const EditExternalClaim: FunctionComponent<EditExternalClaimsPropsInterfa
...
+        return addedClaim.claimURI;
+    };
+
+    /**
+     * Set the modified claimURI to the form values.
+     *
+     * @param {Map<string, FormValue>} values - Claim object
+     */
+    const resolveClaimURI = (values: Map<string, FormValue>): Map<string, FormValue> => {
+        const parts = addedClaim.claimURI.split(":");
```
Comment (brionmario) [inline apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx:187]: Lets add types.

```
const parts: string[] = addedClaim.claimURI.split(":");
```

Comment (kayathiri4) [reply apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx:187]: Fixed with https://github.com/wso2/identity-apps/pull/2574

File: apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx (hunk 1 above)
Comment (kayathiri4) [reply apps/console/src/features/claims/components/edit/external-dialect/edit-external-claim.tsx:174]: Fixed with https://github.com/wso2/identity-apps/pull/2574

**Pull request merged**